merkle/map_hasher_test.go

```diff
@@ -0,0 +1,27 @@
+package merkle
+
+import (
+	"bytes"
+	"encoding/base64"
+	"testing"
+
+	"github.com/google/trillian"
+)
+
+const emptyMapRootB64 = "xmifEIEqCYCXbZUz2Dh1KCFmFZVn7DUVVxbBQTr1PWo="
```

Comment:
I've started adding comments on how to generate them so people can double check the value.